mixins/test/arrow-keys-mixin.html

```diff
@@ -1,157 +1,166 @@
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="/node_modules/mocha/mocha.js"></script>
+		<script src="/node_modules/chai/chai.js"></script>
+		<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+		<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+		<script type="module" src="../demo/arrow-keys-test.js"></script>
+	</head>
+	<body>
 
-<head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-	<title>ArrowKeysMixin test</title>
-	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-	<script src="/node_modules/mocha/mocha.js"></script>
-	<script src="/node_modules/chai/chai.js"></script>
-	<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
-	<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
-	<script type="module" src="./arrow-keys-test-elem.js"></script>
-</head>
-
-<body>
-	<test-fixture id="simpleFixture">
-		<template>
-			<arrow-keys-test-elem></arrow-keys-test-elem>
-		</template>
-	</test-fixture>
-
-	<div dir="rtl">
-		<test-fixture id="simpleFixtureRtl">
+		<test-fixture id="simpleFixture">
 			<template>
-				<arrow-keys-test-elem></arrow-keys-test-elem>
+				<d2l-test-arrow-keys></d2l-test-arrow-keys>
 			</template>
 		</test-fixture>
-	</div>
 
-	<script type="module">
-		import * as helper from '../../helpers/focus.js';
+		<div dir="rtl">
+			<test-fixture id="simpleFixtureRtl">
+				<template>
+					<d2l-test-arrow-keys></d2l-test-arrow-keys>
+				</template>
+			</test-fixture>
+		</div>
 
-		function keyDown(element, keycode) {
-			const event = new CustomEvent('keydown', {
-				detail: 0,
-				bubbles: true,
-				cancelable: true,
-				composed: true
-			});
-			event.keyCode = keycode;
-			event.code = keycode;
-			element.dispatchEvent(event);
-		}
-
-		describe('d2l-focusable-arrowkeys-behavior', () => {
-			let simpleFixture, focusables;
-			const testKeyInteractions = (keyInteractions) => {
-				for (let i = 0; i < keyInteractions.length; i++) {
-					(function(i) {
-						it(keyInteractions[i].name, (done) => {
-							simpleFixture.arrowKeysOnBeforeFocus = (elem) => {
-								return new Promise((resolve) => {
-									expect(elem).to.equal(focusables[keyInteractions[i].endIndex]);
-									resolve();
+		<script type="module">
+			import { getComposedActiveElement } from '../../helpers/focus.js';
+
+			function keyDown(element, keycode) {
+				const event = new CustomEvent('keydown', {
+					detail: 0,
+					bubbles: true,
+					cancelable: true,
+					composed: true
+				});
+				event.keyCode = keycode;
+				event.code = keycode;
+				element.dispatchEvent(event);
+			}
+
+			describe('ArrowKeysMixin', () => {
+
+				let simpleFixture, focusables;
+
+				const testKeyInteractions = (keyInteractions) => {
+					for (let i = 0; i < keyInteractions.length; i++) {
```

Comment:
Not sure if a `forEach` might solve the scoping issue here?

Reply:
Good call, that cleans things up a bit more.